src/Private/Schedules.ps1

```diff
@@ -105,7 +106,7 @@ function Start-PodeScheduleRunspace {
         }
     }
 
-    Add-PodeRunspace -Type Main -ScriptBlock $script -NoProfile
+    Add-PodeRunspace -Type Main -Name 'ScheduleHouseKeeper' -ScriptBlock $script -NoProfile
```

Comment:
This runspace isn't for the housekeeper, as that's further up as a Timer. This is for the scheduler for Schedules, so we could either call it `ScheduleScheduler` (😂), or just `Schedules`.

Reply:
You are right I missed it